pkg/storage/system_intake_test.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 // NewSystemIntake should provide a saveable intake with all struct fields
-func NewSystemIntake() models.SystemIntake {
+func NewSystemIntake(euaID string) models.SystemIntake {
```

Comment:
`testhelpers.RandomEUAID()` might help with this rather than having to manually select new EUA IDs that don't collide